i18n/zh.json

```
@@ -77,78 +77,78 @@
-		"audit_reason": "Verified as \"%s\"",
-		"user_missing": "The wiki user \"%s\" doesn't exist.",
-		"user_missing_reply": "your linked wiki user \"%s\" doesn't exist.",
-		"user_blocked": "**The wiki user %s is blocked!**",
-		"user_blocked_reply": "your linked wiki user **\"%s\" is blocked!**",
-		"user_gblocked": "**The wiki user %s is globally blocked!**",
-		"user_gblocked_reply": "your linked wiki user **\"%s\" is globally blocked!**",
-		"user_disabled": "**The wiki user %s is disabled!**",
-		"user_disabled_reply": "your linked wiki user **\"%s\" is disabled!**",
-		"user_failed": "Discord user %1$s doesn't match the wiki user %2$s.",
-		"user_failed_reply": "your Discord tag doesn't match the wiki user \"%s\".",
-		"user_matches": "Discord user %1$s matches the wiki user %2$s, but doesn't meet the requirements for any roles.",
-		"user_matches_reply": "your Discord tag matches the wiki user \"%s\", but you don't meet the requirements for any roles.",
-		"user_verified": "Discord user %1$s has been successfully verified as wiki user %2$s.",
-		"user_verified_reply": "you have been successfully verified as wiki user \"%s\".",
-		"user_renamed": "Their Discord nickname has been changed to their wiki username.",
-		"discord": "Discord user:",
-		"wiki": "Wiki user:",
-		"empty": "*empty*",
-		"notice": "Notice:",
-		"qualified": "Qualified for:",
-		"qualified_error": "Qualified for, but could not add:",
-		"help_guide": "Follow [this guide](%s) to add your Discord tag to your wiki profile.",
+		"error": "由于一个错误，验证失败。",
+		"error_reply": "由于一个错误，验证失败，请重试。",
+		"missing": "本频道无可用的验证方式。",
+		"footer": "Wiki账户验证",
+		"failed_gblock": "**全域封禁检查失败！**",
+		"failed_roles": "**添加身份组失败！**",
+		"failed_rename": "**更改昵称失败！**",
+		"audit_reason": "已以“%s”的身份验证。",
+		"user_missing": "Wiki用户“%s”不存在。",
+		"user_missing_reply": "您连接的wiki用户“%s”不存在。",
+		"user_blocked": "**Wiki用户 %s 已被封禁！**",
+		"user_blocked_reply": "您连接的wiki用户**“%s”已被封禁！**",
+		"user_gblocked": "**Wiki用户 %s 已被全域封禁！**",
+		"user_gblocked_reply": "您连接的wiki用户**“%s”已被全域封禁！**",
+		"user_disabled": "**Wiki用户 %s 已被禁用！**",
+		"user_disabled_reply": "您连接的**“%s”的账号已被禁用！**",
+		"user_failed": "Discord用户%1$s无法和wiki用户 %2$s 对应。",
+		"user_failed_reply": "您的Discord用户名不和“%s”的对应。",
+		"user_matches": "Discord用户 %1$s 和wiki用户 %2$s 对应，但没有达到任何身份组的条件。",
+		"user_matches_reply": "您的Discord用户名和wiki用户“%s”对应，但您没有达到添加任何身份组的条件。",
+		"user_verified": "Discord用户 %1$s 已成功验证为wiki用户 %2$s。",
+		"user_verified_reply": "您已被成功验证为wiki用户“%s”",
+		"user_renamed": "他们的Discord昵称已被更改为他们的wiki用户名。",
+		"discord": "Discord用户：",
+		"wiki": "Wiki用户：",
+		"empty": "*空白*",
+		"notice": "注意：",
+		"qualified": "符合：",
+		"qualified_error": "符合，但无法添加：",
+		"help_guide": "请根据[此指导](%s)来添加您的Discord用户名至您的wiki资料页。",
 		"help_gamepedia": "https://help.gamepedia.com/Discord",
-		"help_fandom": "https://community.fandom.com/wiki/Special:VerifyUser"
+		"help_fandom": "https://community.fandom.com/zh/wiki/Help:Profiles/Discord"
```

```suggestion
		"help_fandom": "https://community.fandom.com/zh/wiki/Special:VerifyUser"
```
Please use Special:VerifyUser as the script allows for simple addition of the correct Discord tag.